app/model/blockchain/token.py

```diff
@@ -392,7 +394,8 @@ def fetch_expiry_short(self) -> None:
         token_contract = Contract.get_contract("IbetShare", self.token_address)
 
         # Fetch
-        company = CompanyList.get_find(to_checksum_address(self.owner_address))
+        owner_address = Contract.call_function(token_contract, "owner", (), ZERO_ADDRESS)
```

Comment:
need self.owner_address to update

Reply:
Thank you. I've fixed in below commit.

https://github.com/BoostryJP/ibet-Wallet-API/blob/ae1a322342ba85f51cc99bbc2dd179fbd41b072b/app/model/blockchain/token.py#L413-L415

https://github.com/BoostryJP/ibet-Wallet-API/pull/1205/commits/ae1a322342ba85f51cc99bbc2dd179fbd41b072b